seed/tests/data/property_sample_data.json

```diff
@@ -225,4 +225,27 @@
   "owner_city_state": "Colorado",
   "owner_postal_code": 80202,
   "organization_id": -1
+}, {
+  "jurisdiction_property_id": 123481,
```

Comment:
Your new property is getting merged with another one...
<img width="365" alt="Screen Shot 2021-11-08 at 9 43 45 AM" src="https://user-images.githubusercontent.com/18518728/140783209-d63fadc1-ee68-4792-8e1e-c5e9a2ce9b8b.png">

According to the "autopopulate modal" the two properties above your new one are supposed to be merged. Could you check the logic in the endpoint to make sure we're merging the correct properties? My guess is it's just grabbing the last two properties but that wouldn't be right now